_studio/mfx_lib/encode_hw/h265/include/mfx_h265_encode_hw_utils.h

```diff
@@ -473,7 +462,7 @@ namespace ExtBuffer
 
     #define _CopyPar(dst, src, PAR) dst.PAR = src.PAR;
     #define _CopyPar1(PAR) _CopyPar(buf_dst, buf_src, PAR);
-    #define _CopyStruct1(PAR) Copy(buf_dst.PAR, buf_src.PAR);
+    #define _CopyArray(PAR) std::copy(std::begin(buf_src.PAR), std::end(buf_src.PAR), std::begin(buf_dst.PAR));
```

Comment:
These macro defs has a limited life scope. You _must_ adjust line 572 below with `#undef _CopyStruct1`. To be honest, I don't like this code. I would prefer to have explicit code and drop these macro defs completely. @mgonchar : how do you think?